docker/serve/Dockerfile

```diff
@@ -1,15 +1,26 @@
-ARG PYTORCH="1.6.0"
-ARG CUDA="10.1"
-ARG CUDNN="7"
+ARG PYTORCH="1.9.0"
+ARG CUDA="11.1"
+ARG CUDNN="8"
 FROM pytorch/pytorch:${PYTORCH}-cuda${CUDA}-cudnn${CUDNN}-devel
 
-ARG MMCV="1.3.8"
-ARG MMSEGMENTATION="0.14.1"
-ARG MMDET="2.14.0"
-ARG MMDET3D="0.17.1"
+ARG MMCV="2.0.0rc1"
+ARG MMDET="3.0.0rc0"
```

Comment:
please check the mmdet version

Reply:
mmdet version has been updated to latest version